### DISCUSSION TITLE
Get imported emails with the API

### DISCUSSION THREAD

**Discussion by jdhitsolutions** [buttondown/discussions#23] [Q&A] [answered]
I'm trying to understand the API docs. It looks like there is a parameter I can use to see imported emails but I can't get anything to work. The best I can do is use `https://api.buttondown.email/v1/emails` but this only gives me emails I've sent since joining the platform. How can I get my imported emails? A curl example would help.

**Comment by jdhitsolutions** [comment, marked as answer]
Gosh darn it, but details matter!

I was using `https://api.buttondown.email/v1/emails/?status=imported` when I should have used `https://api.buttondown.email/v1/emails?status=imported`

More coffee.

**Reply by jmduke** [reply]
We've got a disclaimer on this in the docs, but I think I need to increase the font size ;)
